src/coresyf_image_crop/coresyf_image_crop.py

```diff
@@ -101,6 +101,24 @@ def apply_buffer_to_polygon(polygon_extent, buffer, crs_polygon, crs_buffer):
     return polygon_with_buffer
 
 
+def get_raster_resolution(raster_path):
+    '''
+    Retrieves the resolution of the raster located at raster_path,
+    in meters/pixel or degress/pixel.
+    '''
+    gdal_raster = gdal.Open(raster_path)
+    transform = gdal_raster.GetGeoTransform()
+    return transform[1]
```

Comment:
Please ensure that the resolution is always a positive value.

Reply:
Done.